doc/guile-libnotify.texi

```diff
@@ -116,7 +117,128 @@ If you wish to install you can @command{make install}, too. It may require
 sudo.
 
 @node How to use
-@chapter How to use
+@chapter How to use Guile-Libnotify
```

Comment:
This chapter is called `How to use` in the menu. The chapter should be consistent with the menu.